src/control/actor/host/notify.go

```diff
@@ -60,14 +60,28 @@ func (c *HostNotifyCmd) connectedF(net network.Network, conn network.Conn) {
 
 	// request metadata as soon as we connect to a peer
 	peerData, err := PollPeerMetadata(conn.RemotePeer(), c.Base, c.PeerMetadataState, c.Store, c.PeerStore)
+	// request BeaconStatus metadata as we connect to a peer
+	h, err := c.Host()
+	if err != nil {
+		log.Error(err)
+	}
+	bState, err2 := nodemetadata.ReqBeaconStatus(context.Background(), h, conn.RemotePeer())
+	if err2 != nil {
+		log.Warn(err)
+	}
+	logrus.Info(&bState)
```

Comment:
Lets add logs a bit more meaningful, containing more information. Rebase to the latest #15 and you will see some examples here in `notify.go`